interview-questions.md

```diff
@@ -8,48 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
-
-Researched answer:
+Your answer: Javascript Syntax eXchange? I forget what the acronym means but it's what allows javascript to play with HTML's toys.
 
+Researched answer: Dang, I was close. The acronym is Javascript Syntax eXtension, or just Javascript XML. An interesting Syntactical difference would be the need to dance around Javascript's reserved keywords, like how "for" in HTMLspeak would be "HTMLfor".
+***something about the phrase "syntactic sugar" really irks me...***
 1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: A package manager, similar to NPM, that we use to install/run React and Jest.
+
+***yarn is a long continuous length of interlocked fibres, used in sewing, crocheting, knitting, weaving, embroidery, ropemaking, and the production of textiles. Yarn is the package manager. gotta watch that Capitalization!***
 
-Researched answer:
+Researched answer: running [yarn] in my terminal gives me the yarn.lock file and a node_modules folder.
 
 3. What is a React component?
 
-Your answer:
+Your answer: a react component is a bit of code in its' own file, made to be reuseable in different applications. say you have a pretty box or CSS animation that you like, If you create it as a react component you can use it wherever and whenever you like!
+**I'd say you could probably make a decent side hustle out of creating quality component libraries, but all these programmers just wanna "share" their code for "free" because "an open web is a better web". Dang cyber-hippies!**
 
-Researched answer:
+Researched answer: What my prior answer says, but way smarter and more official sounding.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: state values can change, props are static.
 
-Researched answer:
+Researched answer: Ok so apparently this topic is way more complex and technical than I realized, and there are a ton of fiddly little differences between the two, but in general a component's prop(ertie)s are much more static than its state values.
 
 5. What is the DOM?
 
-Your answer:
-
-Researched answer:
-
+Your answer: Blind guess on the acronym; Document Object Manager? Regardless of what it's called, The DOM is like a transit hub, handling the import/exporting of components inside the react application.
+***...heh...***
+Researched answer: On the right track, but not quite deep enough. the Documents Object Model isn't quite like a Transit Hub, but more like a deployment strategy. It's what handles changes to the UI on a page, updating based on user input. React brings a Virtual DOM, greatly improving this method by internally breaking the app into a 'tree' of components, only updating whatever is necessary instead of the whole app whenever an individual element is manipulated.
```

Comment:
Yeah, I love that you followed up with this question and really dove into it with the backed researched answer. This is awesome!